aries_cloudagent/core/dispatcher.py

```diff
@@ -170,7 +175,7 @@ async def handle_message(
         # if warning:
```

Comment:
Is there a reason for this code block to be commented-out rather than removed?

Reply:
The commented block was added with https://github.com/hyperledger/aries-cloudagent-python/pull/1940, it supports handling of minor version of protocols. This implements what was mentioned in the RFC but a decision was made at that time to comment this out and reconsider in future.